### ISSUE TITLE
Can't use Bitstamp

### ISSUE THREAD

**Issue by vea45** [DeviaVir/zenbot#1288]
I can't start Zenbot using any form of the "Zenbot.sh trade bitstamp.xxx-xxx" because it can't backfill data and blocks the execution...

**Comment by araviel** [comment]
I agree, it won't trade at all even without backfilling, like bittfinex

`osboxes@osboxes:~/zenbot$ ./zenbot.sh trade bitstamp.xrp-usd
fetching pre-roll data:

cannot backfill bitstamp: exchange does not offer historical data
osboxes@osboxes:~/zenbot$ `

**Comment by vea45** [comment]
Now the backfilling problem is solved but I can't still trade anything with Bitstamp...

**Comment by vithonch** [comment]
I see buy/sell decisions but no real API requests happen. Any ideas?

**Comment by JNTNX433** [comment]
Same here

**Comment by vithonch** [comment]
Seems the buy/sell commands doesn't work as well

**Comment by vea45** [comment]
Nothing is working at all... the only thing that is working is saving the data to the DB.
